src/common/types/contentPreview.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+import * as React from 'react';
+
+import ContentPreview from '../../elements/content-preview';
+
+export type ContentPreviewProps = $Diff<
```

Comment:
Can we clarify why this is needed? Can we import the props directly wherever ContentPreview is being used, since that location will already have an import?

Reply:
It does make sense for me to move it to `MessagePreviewContent`.  I added it to the file I'm not sure if we want to keep the type there or I should rather create a dedicated file for it. What is you opinion?